package.json

```diff
@@ -6,6 +6,8 @@
         "puppeteer": "^10.4.0",
         "serve-handler": "^6.1.3"
     },
+    "dependencies": {
+        "codemirror": "^5.63.1"
     "devDependencies": {
```

Comment:
Codemirror doesn't not neet to be a dependency here (since it will not be used by CLI).

Also, this package.json is malformed.